hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/__tests__/locators/locators.ts

```diff
@@ -16,12 +16,25 @@
  * limitations under the License.
  */
 
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './app';
+export const overviewLocators = {
+  "datanodeCard": "overview-Datanodes",
+  "pipelinesCard": "overview-Pipelines",
+  "clusterCapacityCard": "overview-Cluster Capacity",
+  "volumesCard": "overview-Volumes",
+  "bucketsCard": "overview-Buckets",
+  "keysCard": "overview-Keys",
+  "deletedContainersCard": "overview-Deleted Containers",
+  "openKeysSummaryCard": "overview-Open Keys Summary",
+  "deletePendingSummaryCard": "overview-Pending Deleted Keys Summary"
+}
```

Comment:
Are these going to be mapped to `data-testid`? I can't seem to find these for overview page.

Reply:
Thanks for pointing this out, updated in the latest commit